screengrab/build.gradle

```diff
@@ -1,24 +1,25 @@
-// Top-level build file where you can add configuration options common to all sub-projects/modules.
+apply plugin: 'com.github.ben-manes.versions'
 
 buildscript {
     repositories {
-        jcenter()
+        google()
+        mavenCentral()
+        gradlePluginPortal()
```

Comment:
new best practice?

Reply:
Yes. jcenter is often flaky or mirrors something that fails when being downloaded due to wrong checksums.

`google()` is the way to go. `mavenCentral` contains all of the things and `gradlePluginPortal()` is for the newly introduced plugin.